### ISSUE TITLE
Content overlaps at 200% text enlarge

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15357]
Content overlaps at 200% text enlarge

-   **Severity:** Medium
-   **Affected Populations:**
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -  Structure and Navigation

#### Issue description

The summary text at the top of the Content Structure popup is unreadable
at 200% text enlarge, due to overlapping text. The containers have set
widths and cannot adjust to changes in font-size, which causes
individual words (which cannot wrap without hard word-breaks) to
overflow their containers.

The ability to resize text is essential for users with low-vision, and
may be helpful for users who have a cognitive disability. Catering to
zoom alone is not sufficient because the browser's font-size may be
increased independently of zoom level.

##### Issue Code

```
    .table-of-contents__count {


        width: 25%;


        display: flex;


        flex-direction: column;


        ...


    }

```

#### Remediation Guidance

Avoid fixed widths on containers (even percentage-based widths) when the
content inside cannot wrap.

Instead, set min-widths on containers, and allow them to expand with
increases in content size, allowing the containers themselves to wrap as
needed.

#### Relevant standards

-   [1.4.4 Resize text (Level
    AA)](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-scale)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-21 in Tenon's report

**Comment by melchoyce** [comment]
Full report screenshot:

![image](https://user-images.githubusercontent.com/2846578/57186899-e146a380-6e9b-11e9-85aa-4198f6453db3.png)

**Comment by afercia** [comment]
Discussed during today's accessibility bug scrub. Not so much to add here, just needs to be fixed 🙂 

Specifically for the document outline, it should not have a fixed width and on small screens should take all the available width to make room for content.

We'd guess also other components needs to have their layout checked in this scenario.
